python/spacewalk/server/importlib/headerSource.py

```diff
@@ -289,6 +289,9 @@ def _populateTag(self, header, tag, Class):
                 # pylint: disable-next=use-implicit-booleaness-not-len
                 elif not len(v) and k in ("device", "flags"):
                     hash[k] = 0
+                # file size information is empty string when not available
+                elif not len(v) and k in ("file_size", "file_size_long"):
```

Comment:
Linting's not happy, you probably want:

```suggestion
                elif not v and k in ("file_size", "file_size_long"):
```

Reply:
Alternatively, explicit bool check:

```suggestion
                elif not bool(v) and k in ("file_size", "file_size_long"):
```

Reply:
Another yet alternative, explicit len check:

 ```suggestion
                elif len(v) == 0 and k in ("file_size", "file_size_long"):
```